### ISSUE TITLE
App cleanup is not called in linux

### ISSUE THREAD

**Issue by gaborpapp** [cinder/Cinder#1404]
`cleanup()` does not seem to be called on exit.

I added the following line after the loop in `AppImplLinuxGlfw.cpp` to fix it:

    mApp->emitCleanup();

**Comment by PetrosKataras** [comment]
True, this seems to be missing currently from both the GLFW and the EGL app implementations..
